CondCore/CTPPSPlugins/test/testDAQMapping.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [ "$1" == "run" ]
+then
+    mkdir -p CondCore/CTPPSPlugins/test/results
+    if [ -f *.png ]; then    
+    rm *.png
+    fi
+
+    echo "Testing DAQMapping info"
+
+    python3 CondCore/Utilities/scripts/getPayloadData.py \
```

Comment:
```suggestion
    getPayloadData.py \
```

this presumes that the one has checked out ` CondCore/Utilities` in the first place.
`getPayloadData.py ` is a script avalailble in the path.

Reply:
@Kszemka looks like this suggestion was not applied. Any reason for that?

Reply:
accidentally missed, I already updated. c: